### ISSUE TITLE
Tmpfiles - set extended attribute

### ISSUE THREAD

**Issue by mvorisek** [systemd/systemd#13151]
I need to set extended file attribute like:
```
root@debian:~# lsattr -l /tmp/test/.htaccess
/tmp/test/.htaccess          Immutable, Extents
```

I tried:
```
F  "/tmp/test/.htaccess"  0644  root  root  -  "Require all denied"
t  "/tmp/test/.htaccess"  -     -     -     -  "Immutable"
```
but when I run the `systemd-tmpfiles --create --remove` command, it outputs:
```
Failed to parse extended attribute, ignoring: Immutable
```

What is the correct way to set the `Immutable` extended file flag or is it an systemd issue?

**Comment by poettering** [comment]
extended attributes (as managed by setfattr/getfattr) are different from the additional file attribute bits that chattr/lsattr manipulate.

For extended attributes use "t" lines. For the additional file attributes use "h". See documentation.

I guess we could extend the documentation to mention the possible confusion explicitly, after all we manage both.

**Comment by poettering** [comment]
(make a documentation bug hence)

**Comment by poettering** [comment]
Fix waiting in #13165
